rero/rero-ils
# pids: cli to reserve a range of pids

* Assigns all unused pids the status NEW.
* Assigns all reserved pids the status RESERVED.
* Improves method to insert identifer pids and reset sequence.

Co-Authored-by: Aly Badr <aly.badr@rero.ch>

## How to test?

-  script/setup
- make sure sequences are set correctly (pgAdmin)

- for information how to use the new cli:
pipenv run invenio utils reserve_pid_range --help

- test the cli and reserve a range of pids and consult the console to check the status of the reserved pids.

- an example to reserve 10 pids for patron future loading and assign status NEW to all unused pids:
`pipenv run invenio utils reserve_pid_range -p ptrn -n 10 -u`

## Code review check list
